pkg/agent/sysadvisor/plugin/qosaware/resource/cpu/region/headroompolicy/policy_canonical.go

```diff
@@ -55,6 +55,14 @@ func (p *PolicyCanonical) Update() error {
 			return err
 		}
 
+		if p.regionType == types.QoSRegionTypeDedicatedNumaExclusive && !enableReclaim {
```

Comment:
should this be moved out of range loop for `for podUID, containerSet := range p.podSet {`? since you will add  up multiple times for `cpuEstimation` and `containerCnt` this way

Reply:
and the same for `provision_canonical.go`